Marek Mahut <mmahut@redhat.com>: making service now MID server SysVInit aware

diff -Naur servicenow.orig/agent/bin/mid.sh servicenow/agent/bin/mid.sh
--- servicenow.orig/agent/bin/mid.sh    2012-09-10 09:22:08.000000000 -0400
+++ servicenow/agent/bin/mid.sh 2012-09-10 09:26:03.000000000 -0400
@@ -1,4 +1,18 @@
-#! /bin/sh
+#!/bin/sh
+#
+# servicenow-mid Service Now MID server
+#
+# chkconfig:   235 30 70
+# description: servicenow-mid
+#              
+
+### BEGIN INIT INFO
+# Provides: servicenow-mid
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+# Short-Description:  start and stop servicenow-mid
+# Description: service now mid server     
+### END INIT INFO
 
 #
 # Copyright (c) 1999, 2006 Tanuki Software Inc.

